scripts/rook.sh

```diff
@@ -1,14 +1,33 @@
 #!/bin/bash -e
 
-ROOK_VERSION=${ROOK_VERSION:-"v1.1.7"}
+ROOK_VERSION=${ROOK_VERSION:-"v1.2.7"}
```

Comment:
This should probably be placed in `/build.env` ? Not required, but would be considered a best practice while updating scripts like this,

Reply:
yeah. sure.

Reply:
do you plan to update this PR with that recommendation?